docs/content/users/details/offline-usage.md

```diff
@@ -13,7 +13,7 @@ You have general options as well:
 
 In `.ddev/config.yaml`, `use_dns_when_possible: false` will make DDEV never try to use DNS for resolution, instead adding hostnames to `/etc/hosts`. You can also use `ddev config --use-dns-when-possible=false` to set this configuration option.
 
-In `.ddev/config.yaml`, `project_tld: example.com` (or any other domain) can set DDEV to use a project that could never be looked up in DNS. You can also use `ddev config --project-tld=example.com`.
+In `.ddev/config.yaml` `project_tld: example.com` (or any other domain) can set DDEV to use a project TLD that could never be looked up in DNS. You can also use `ddev config --project-tld=example.com`. This also works as a global option in `~/.ddev/global_config.yaml` or use `ddev config global --project-tld=example.com`.
```

Comment:
I don’t think you have to explain that `example.com` is a placeholder, and it might be good to give a _little_ context to the difference between `.ddev/config.yaml` and `project_tld: example.com` instead of just a space:

```suggestion
In `.ddev/config.yaml`, you can use `project_tld: example.com` to have DDEV use a project TLD that won’t be looked up via DNS. You can do the equivalent with `ddev config --project-tld=example.com`. This also works as a global option in `~/.ddev/global_config.yaml` or running `ddev config global --project-tld=example.com`.
```